src/Diagnostics.RuntimeHost/Services/ResourceService/StampService.cs

```diff
@@ -104,7 +104,7 @@ private string GetTenantIdQuery(string stamp, DateTime startTime, DateTime endTi
             return
                 $@"{tableName}
                 | where TIMESTAMP >= datetime({startTimeStr}) and TIMESTAMP <= datetime({endTimeStr})
-                | where PublicHost =~ ""{stamp}.cloudapp.net"" or PublicHost matches regex ""{stamp}([a-z{{1}}]).cloudapp.net""
+                | where PublicHost =~ ""{stamp}.cloudapp.net"" or PublicHost matches regex ""{stamp}([a-z{{1}}]).cloudapp.net"" or EventStampName =~ ""{stamp}""
```

Comment:
Just do a quick check for:

- **Mega stamps** - I think there is another column called 'EventPrimaryStampName' that will hold the mega stamp name and not this column. This 'EventStampName' would have something like 'waws-prod-sn1-107a'
Ex:
```
AntaresIISLogFrontEndTable
| where TIMESTAMP > ago(20m) and EventStampName  == 'waws-prod-sn1-107a'
| summarize by EventStampName, EventPrimaryStampName
```

  |   |  
-- | -- | --
  | **EventStampName** | **EventPrimaryStampName**
  | waws-prod-sn1-107a | waws-prod-sn1-107



- **Sites in ASE** - Make sure you are seeing the right name in this column which is passed in the stamp parameter to this method and there is no friendly name issue.